modules/ROOT/partials/aws-s3-credentials.adoc

```diff
@@ -21,9 +20,12 @@ The output should look like this:
 [result,shell]
 ----
 [default]
-region=eu-north-1
+region=us-east-1
 ----
-.. Verify that the `~/.aws/credentials` is correct:
+. Configure the access to your AWS S3 bucket by either setting the `aws_access_key_id` and `aws_secret_access_key` in the `~/.aws/credentials` file or by using a bucket policy.
```

Comment:
You'll always need something like the `aws_access_key_id` and `aws_secret_access_key` for authentication

The bucket policy is more for authorisation once your auth'd in the system